src/main/main.cpp

```diff
@@ -48,8 +54,13 @@ int main(int argc, char *argv[])
 
     QGuiApplication app(argc, argv);
 
+    #ifdef Q_OS_MACOS
+    MacOsEventListener evListener;
+    initMacOsEventListener(&evListener);
```

Comment:
@tim-sueberkrueb not sure what do you mean by intend all compiler directives. Could you elaborate more?

This code is Mac OS X dependent. Thus conditional compilation.